pythonforandroid/bootstraps/sdl2/build/jni/src/start.c

```diff
@@ -156,7 +156,7 @@ int main(int argc, char *argv[]) {
   if (dir_exists("lib")) {
     /* If we built our own python, set up the paths correctly */
     LOGP("Setting up python from ANDROID_PRIVATE");
-    PyRun_SimpleString("private = posix.environ['ANDROID_PRIVATE']\n"
+    PyRun_SimpleString("private = posix.environ['ANDROID_PRIVATE'] + '/app'\n"
```

Comment:
This should actually use ANDROID_APP_PATH rather than guessing /app, right? If that's right, just let me know and I'll make the change manually and merge.